package.json

```diff
@@ -94,7 +94,7 @@
     "tmp-promise": "^3.0.2"
   },
   "engines": {
-    "node": ">=8.3.0"
+    "node": ">=10.3.0"
```

Comment:
I was torn on whether to use `10.0.0`. which we technically should support, or `10.3.0`, the lowest version we test against in the CI. @ehmicky what do you think?

Reply:
I answered in the other comment. :+1: